GCRCatalogs/register.py

```diff
@@ -142,6 +159,17 @@ def root_dir(self, path):
             warnings.warn("root_dir has been set to '{}' but errors may occur when you try to access it: {}".format(path, e))
         self._custom_root_dir = path
 
+    def set_root_dir_by_site(self, site):
+        """
+        If *site* is a recognized site, set root_dir to corresponding value
+        """
+        if site in self._site_list:
+            self.root_dir = self._site_config[site]
+        else:
+            site_string = ' '.join(_config_register._site_list)
```

Comment:
Use the new public `site_list` property
```suggestion
            site_string = ' '.join(_config_register.site_list)
```